docs/recipes/pipelines.ipynb

```diff
@@ -13,7 +13,9 @@
    "source": [
     "Pipelines are an integral part of River. We encourage their usage and apply them in many of their examples.\n",
     "\n",
-    "The `compose.Pipeline` contains all the logic for building and applying pipelines. A pipeline is essentially a list of estimators that are applied in sequence. The only requirement is that the first `n - 1` steps be transformers. The last step can be a regressor, a classifier, a clusterer, a transformer, etc. Here is an example:"
+    "The `compose.Pipeline` contains all the logic for building and applying pipelines. A pipeline is essentially a list of estimators that are applied in sequence. The only requirement is that the first `n - 1` steps be transformers. The last step can be a regressor, a classifier, a clusterer, etc.\n",
```

Comment:
QQ: Does it make sense to have a transformer-like as last step? From my understanding it doesn't as:
- it would not be called in `transform_one`
- it would break any `predict_*` method